spisea/imf/multiplicity.py

```diff
@@ -194,7 +199,10 @@ def random_companion_count(self, x, CSF, MF):
         Helper function: calculate number of companions.
         """
         n_comp = 1 + np.random.poisson((CSF / MF) - 1)
-
+        
+        if self.companion_max == True:
```

Comment:
I'm a bit confused here. Isn't this saying that n_comp is always set to companion_max if self.companion_max? Shouldn't it only set to companion_max if n_comp is greater than self.companion_max?

Also, isn't self.companion_max a boolean, based on the if statement? Then it isn't actually a number, and ultimately n_comp needs to be a number.

Reply:
Sorry that was an error. It should be fixed now!